SEED-platform/seed
# 3218 feat/refresh metadata property state

#### Any background context you want to provide?
The `refresh_metadata` feature was requested for use with the OEP Connection where the `Property.updated` attribute is used for filtering. While this feature was functional there was no UI indication that anything had successfully occurred.

#### What's this PR do?
In addition to updating a Property's (or Taxlot's) updated attribute, the `refresh_metadata` feature now updates the related PropertyState's (or TaxlotState's) updated attribute and can be validated when viewing the inventory detail.

#### How should this be manually tested?
- Go to Inventory 
- make note of the 'updated' column 
- select one or more rows 
- Actions > Refresh metadata 
- confirm 'updated' values are 'now' 

#### What are the relevant tickets?
#3128

#### Screenshots (if appropriate)

https://user-images.githubusercontent.com/58446472/172206007-f3813925-3807-4b75-92d4-e8ceac8a0797.mov
